src/DependencyInjection/WebpackEncoreExtension.php

```diff
@@ -51,6 +51,11 @@ public function load(array $configs, ContainerBuilder $container)
         $container->getDefinition('webpack_encore.entrypoint_lookup_collection')
             ->replaceArgument(0, ServiceLocatorTagPass::register($container, $factories));
         $container->setAlias(EntrypointLookupInterface::class, new Alias($this->getEntrypointServiceId('_default')));
+
+        $container->getDefinition('webpack_encore.tag_renderer')
+            ->replaceArgument(2, [
+                'crossorigin' => $config['crossorigin'],
```

Comment:
I would rather filter out `false` value here at compile time rather than filtering them at runtime.

Reply:
I think I have implemented this like you recommend. ok?